doc/cephadm/install.rst

```diff
@@ -374,7 +374,8 @@ Deployment in an isolated environment
 -------------------------------------
 
 You can install Cephadm in an isolated environment by using a custom 
-container registry. You can either configure Podman or Docker to use 
+container registry. Sometimes also referred as airgap. 
```

Comment:
Do you want to mention that it's usually considered "airgapped" if that registry is not directly connected to the internet or do you think that's over-complicating things?